packages/arb-token-bridge-ui/src/components/TransferPanel/CctpTabContent.tsx

```diff
@@ -0,0 +1,45 @@
+import { PropsWithChildren } from 'react'
+import { CCTP_DOCUMENTATION } from '../../constants'
+import { useCCTPIsBlocked } from '../../hooks/CCTP/useCCTPIsBlocked'
+import { ExternalLink } from '../common/ExternalLink'
+
+export const CctpTabContent = ({
+  toNetworkName,
+  children
+}: PropsWithChildren<{
+  toNetworkName: string
+}>) => {
+  const { data: isCctpBlocked, isLoading: isLoadingIsCctpBlocked } =
+    useCCTPIsBlocked()
+
+  if (isLoadingIsCctpBlocked || isCctpBlocked) {
+    return (
+      <>
+        Access to Circle&apos;s bridge is restricted in certain jurisdictions.
+        For more details, please consult Circle&apos;s{' '}
+        <ExternalLink
```

Comment:
![image](https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/0b7a07a2-865d-4630-8519-45dbebba65fc)
this anchor link is occupying the whole row which it shouldn't

Reply:
I now wrap the text with a `p` tag, which solves the issue